apisupport.harness/release/suite.xml

```diff
@@ -420,7 +420,17 @@ See: http://wiki.netbeans.org/DevFaqHowToReuseModules
               tofile="${dist.dir}/${app.name}.app/Contents/Resources/${app.name}.icns"/>
         
         <delete file="${dist.dir}/${app.name}.app/Contents/MacOS/${app.name}"/>
-        <symlink link="${dist.dir}/${app.name}.app/Contents/MacOS/${app.name}" resource="../Resources/${app.name}/bin/${app.name}"/>
+        <echo file="${dist.dir}/${app.name}.app/Contents/MacOS/${app.name}">#!/bin/sh
```

Comment:
I am a Linux user, maybe our Mac users @tzezula or @thurka can help with the review.

Reply:
Have they been notified of the request by you referencing them? I haven't noticed anyone checking this change yet. It does work for me, if that means anything.